actors/builtin/multisig/multisig_actor.go

```diff
@@ -32,6 +34,15 @@ type Transaction struct {
 	Approved []addr.Address
 }
 
+type ApprovalHashData struct {
+	MsigWallet addr.Address
```

Comment:
unneeded as the address of the multisig actor is the To of the Approve/Cancel Message, and so is already protected by signing scheme